app/domain/operations/slcsp_calculation.rb

```diff
@@ -56,7 +56,9 @@ def process(params)
           if prev_month_data[:residence] != current_month_data[:residence] && !last_full_result.nil? && !current_month_data[:members].blank? && !current_month_data[:primary_absent]
             @logger.info "SLCSP ------------------------- residence changed"
             # figure it out if we are on a different rating area
-            seeker = calculate_month(current_month_data, params[:taxYear], i, month_key).value!
+            seeker = {}
+            seeker_call = calculate_month(current_month_data, params[:taxYear], i, month_key)
+            seeker = seeker_call.value! if seeker_call.success?
             if last_full_result[:rating_area_id] != seeker[:rating_area_id] && last_full_result[:service_area_ids] != seeker[:service_area_ids]
```

Comment:
This will break if calculate_month does not return success.  Is there any reason we would be positive that it cannot return a failure?  Perhaps we want to bubble up the failure message returned by calculate_month here if it fails?